apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -148,23 +150,31 @@ private void publishEvent(
     }
     if (partition != null) {
       JSONArray partitionValuesArray = new JSONArray(partition.getValues());
```

Comment:
Move this down to line 160 as I think that will be easier to read - first you build up and set the keys, then you set the values. Generally we try keep the declaration of a new variable as close to its first use as possible.